### PR TITLE
chore(demo): migrate documentation page `Breadcrumbs` to standalone components

### PR DESCRIPTION
**Pull request by mdlufy**
Partially solves #6957 
Done: Migrate Breadcrumbs component to standalone

### CODE REVIEW

File: projects/demo/src/modules/components/breadcrumbs/index.ts [hunk 1]
```diff
@@ -1,27 +1,18 @@
...
-import type {TuiSizeL} from '@taiga-ui/core';
+import {TuiDemoModule} from '@demo/utils';
+import {TuiLinkDirective, type TuiSizeL} from '@taiga-ui/core';
+import {TuiBreadcrumbsModule} from '@taiga-ui/kit';
 
 @Component({
-    selector: 'example-breadcrumbs',
-    templateUrl: './breadcrumbs.template.html',
+    standalone: true,
+    imports: [NgFor, RouterLink, TuiDemoModule, TuiBreadcrumbsModule, TuiLinkDirective],
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/breadcrumbs/index.ts:11]: `TuiDemoModule` contains `CommonModule`, so `NgFor` is redundant

File: projects/demo/src/modules/components/breadcrumbs/examples/import/index.html
Comment (vladimirpotekhin) [inline projects/demo/src/modules/components/breadcrumbs/examples/import/index.html:1]: looks like an extra file

Comment (mdlufy) [reply projects/demo/src/modules/components/breadcrumbs/examples/import/index.html:1]: delete

File: projects/demo/src/modules/components/breadcrumbs/index.ts (hunk 1 above)
Comment (mdlufy) [reply projects/demo/src/modules/components/breadcrumbs/index.ts]: update

**Review by vladimirpotekhin — APPROVED** [review]

**Review by splincode — APPROVED** [review]

**Pull request merged**